test/manifest.ts

```diff
@@ -2049,4 +2049,80 @@ describe('Manifest', () => {
       });
     }
   });
+
+  describe('plugins', () => {
+    it('runs the node-workspace plugin', async function () {
```

Comment:
A few additional comments in this test would help me. Shouldn't we see `@pkg2` being updated as well since it depends on `@pkg1` which was updated? I only see updates for `@pkg1` and `python`.

Reply:
if you look at [the snapshot](https://github.com/googleapis/release-please/pull/844/files#diff-aaf3d7c2b71cbe7339707b4a850d46bdc43abdafb291f1a874de7a9c8fef954eR38) it captured the `@pkg2` updates. Perhaps confusing because I'm only checking the logging that `Manifest` is performing here, not any logging the plugin is doing. Given how the plugin is instantiated via the config it's hard to inject a logger (though I could probably stub it). Or I could just remove the `Manifest` logging for this test case to avoid confusion.

I'll get around to moving the logging out of this file into the snapshot too at some point.